test/e2e/common/kubelet_etc_hosts.go

```diff
@@ -57,8 +57,9 @@ var _ = framework.KubeDescribe("KubeletManagedEtcHosts", func() {
 			1. The Pod with hostNetwork=false MUST have /etc/hosts of containers managed by the Kubelet.
 			2. The Pod with hostNetwork=false but the container mounts /etc/hosts file from the host. The /etc/hosts file MUST not be managed by the Kubelet.
 			3. The Pod with hostNetwork=true , /etc/hosts file MUST not be managed by the Kubelet.
+		This test is marked LinuxOnly since Windows cannot mount individual files in Containers.
```

Comment:
Does Windows have /etc/hosts?

Reply:
the equivalent in windows is C:\Windows\System32\drivers\etc\hosts

Reply:
Indeed, it is as @michmike says, but it is not managed by Kubelet because individual files cannot be mounted on Windows Containers.